libs/remix-debug/src/storage/storageResolver.ts

```diff
@@ -133,7 +133,7 @@ export class StorageResolver {
         resolve([{}, null])
       } else {
         this.web3.debug.storageRangeAt(
```

Comment:
does this require change in remix-debug unit tests?

Reply:
I don't believe so, this code should already be touched by remix-debug unit test.